packages/plugins/opentelemetry/package.json

```diff
@@ -52,7 +52,7 @@
   },
   "dependencies": {
     "@envelop/on-resolve": "^4.1.0",
-    "@opentelemetry/api": "^1.0.0",
```

Comment:
It seems that we are using some newer stuff than `1.0.0`.
Not sure what the minimal version should be, so updating to the latest.